components/FormCreate.js

```diff
@@ -13,14 +13,27 @@ import {
   MinusButton,
   ImageContainer,
   SearchImage,
+  TinyInputsWrapper,
+  TinyInput,
+  TinyDiv,
+  ModalContainer,
 } from "./styledComponents/FormCreate.styles";
 import { useRouter } from "next/router";
-import { useState } from "react";
+import { useState, useEffect } from "react";
 import useSWR from "swr";
+import dynamic from "next/dynamic";
+
+const MapGeodata = dynamic(() => import("./MapGeodata"), {
+  ssr: false,
+});
 
 export default function FormCreate() {
+  const [showModal, setShowModal] = useState(false);
   const [searchTerm, setSearchTerm] = useState("");
   const [increment, setIncrement] = useState(0);
+  const [lat, setLat] = useState(null);
```

Comment:
const [lat, setLat], const [lng, setLng] :Please use full word for naming a variable, short hand notation might lead confusion at later point of time